### PR TITLE
Use getCommonGroups not contact

### PR DESCRIPTION
**Pull request by shirser121**
Allow to use getCommonGroups for not contact numbers

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -902,7 +902,13 @@ class Client extends EventEmitter {
      */
     async getCommonGroups(contactId) {
         const commonGroups = await this.pupPage.evaluate(async (contactId) => {
-            const contact = window.Store.Contact.get(contactId);
+            let contact = window.Store.Contact.get(contactId);
+            if (!contact) {
+                const wid = window.Store.WidFactory.createUserWid(contactId);
+                const chatConstractor = window.Store.Contact.getModelsArray().find(c=>!c.isGroup).constructor;
```
Comment (tuyuribr) [inline src/Client.js:908]: Typo, everything else LGTM

**Review by tuyuribr — CHANGES_REQUESTED** [review]
typo

**Comment by shirser121** [comment]
> typo

What should I change?

**Comment by tuyuribr** [comment]
> > typo
> 
> What should I change?

chatConstractor to chatConstructor

**Review by purpshell — APPROVED** [review]
Approved

**Review by tuyuribr — APPROVED** [review]

**Pull request merged**